src/Illuminate/Auth/AuthManager.php

```diff
@@ -63,7 +63,7 @@ public function __construct($app)
      * @param  string|null  $name
      * @return \Illuminate\Contracts\Auth\Guard|\Illuminate\Contracts\Auth\StatefulGuard
      */
-    public function guard($name = null)
+    public function guard($name = null): \Illuminate\Contracts\Auth\Guard|\Illuminate\Contracts\Auth\StatefulGuard
```

Comment:
It's common practice to import classes via `use`
```suggestion
    public function guard($name = null): StatefulGuard
```